packages/web-pkg/src/components/sideBar/Spaces/Details/SpaceDetails.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div id="oc-space-details-sidebar">
     <div class="oc-space-details-sidebar-image oc-text-center">
-      <oc-spinner v-if="loadImageTask.isRunning" />
-      <div v-else-if="spaceImage" class="oc-position-relative">
+      <oc-spinner v-if="showSpaceImage && loadImageTask.isRunning" />
```

Comment:
Wouldn't it make more sense to have an early return in the `loadImageTask` if `showSpaceImage === false`?

Reply:
Then the template can remain unchanged (this line and next line).

Reply:
Alright, since I need to rebase anyway 😄